src/containers/products/ChronicleProductPage.js

```diff
@@ -5,30 +5,148 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import FooterSection from '../footer/FooterSectionWithPageLinks';
+import { Content, PageDescriptionWrapper } from '../../components/layout/IntroComponents';
+import {
+  ExtendedFeatureDescription,
+  ExtendedFeatureHeader,
+  ExtendedFeaturesContent,
+  FeatureShot,
+  Icon,
+  ProductOverviewDescriptionWrapper,
+  ProductOverviewWrapper,
+  ProductShot,
+  ProductStamp,
+} from './styled/StyledProductComponents';
+import { 
+  MEDIA_QUERY_LG,
+  MEDIA_QUERY_MD,
+  MEDIA_QUERY_TECH_SM
+   } from '../../core/style/Sizes';
+import { NEUTRALS } from '../../core/style/Colors';
+import { PageIntro } from '../../components/headers/PageHeaders';
+import ChronicleIcon from '../../assets/logos/placeholder-logo.svg';
+import FeatureImg1 from '../../assets/images/chronicle/chronicle_participants.png';
```

Comment:
This screenshot doesn't fit in the surrounding box, needs resizing.

<img width="586" alt="Screen Shot 2020-04-20 at 2 00 01 PM" src="https://user-images.githubusercontent.com/440299/79799513-66c29e00-830f-11ea-9e6a-5fce4e691dc2.png">

Reply:
did all resizing.